INSTALL.rst

```diff
@@ -181,9 +181,25 @@ new M1 or M2 chips), everything should work natively, so we
 recommend either :ref:`Option 1` or :ref:`Option 2`.
 
 All users on ARM chips, as well as all Windows users, may have to
-use the toolbox within Docker (:ref:`Option 3`), depending on what tools they intend to use.
+take care when installing the toolbox, depending on which tools they
+intend to use.
   
   - The automatic wire cut search in the ``cutqc`` package depends
     on CPLEX, which is only available on Intel chips and is not yet available
     for Python 3.11.
   - The entanglement forging tool requires PySCF, which does not support Windows.
+
+In each case, one method that is guaranteed to work is to :ref:`use
+the toolbox within Docker <Option 3>`.  Other methods include:
```

Comment:
You *could* say within a "container" here to de-emphasize Docker, but again, up to you.